eng/targets/CSharp.Common.props

```diff
@@ -1,7 +1,7 @@
 <Project>
 
   <PropertyGroup>
-    <LangVersion>7.3</LangVersion>
+    <LangVersion>8.0</LangVersion>
```

Comment:
The SDK we have doesn't support "preview" (https://devblogs.microsoft.com/dotnet/an-update-to-c-versions-and-c-tooling) as yet. I think we may want to eventually just use the LangVersion the SDK suggests. In the meanwhile, this seems like a reasonablestop-gap.